src/components/Speaker.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import dummySpeaker from "../images/speakers/halftoneAvatar.png";
+import twitterLogo from "../images/twitter.png";
+import githubLogo from "../images/github.png";
+import websiteLogo from "../images/website.png";
+import matrixLogo from "../images/matrix.png";
+
+function getAlphab(index) {
+  if (index < 0) {
+    return null; // Index out of range for alphabetical characters
```

Comment:
Wonder if we should not throw an error (text the one in the comment) instead of returning null - easier to debug if it really happens